### PR TITLE
RSpec runner and changes to git diff

### PR DESCRIPTION
**Pull request by gmalette**
cc @etiennebarrie

### CODE REVIEW

File: lib/covertrace/runner/rspec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      private
+
+      attr_reader(
+        :dependencies,
+        :merge_base,
+        :root,
+      )
+
+      def tests_to_run
+        changes = Covertrace::GitChanges.changes(merge_base: "origin/master", root: ".")
```
Comment (etiennebarrie) [inline lib/covertrace/runner/rspec.rb]: does that mean it only works if the current working directory is at the root of the git repo?

**Comment by etiennebarrie** [comment]
🚢

File: lib/covertrace/runner/rspec.rb (hunk 1 above)
Comment (gmalette) [inline lib/covertrace/runner/rspec.rb]: lol good catch, those are passed in the initializer :P

Comment (gmalette) [inline lib/covertrace/runner/rspec.rb]: fixed

**Pull request merged**